src/Compilers/CSharp/Portable/Symbols/NamespaceOrTypeSymbol.cs

```diff
@@ -168,26 +170,26 @@ public virtual ImmutableArray<NamedTypeSymbol> GetTypeMembers(string name, int a
         /// Get a source type symbol for the given declaration syntax.
         /// </summary>
         /// <returns>Null if there is no matching declaration.</returns>
-        internal SourceNamedTypeSymbol GetSourceTypeMember(TypeDeclarationSyntax syntax)
+        internal SourceNamedTypeSymbol? GetSourceTypeMember(TypeDeclarationSyntax syntax)
         {
-            return GetSourceTypeMember(syntax.Identifier.ValueText, syntax.Arity, syntax.Kind(), syntax);
+            return GetSourceTypeMember(syntax.Identifier.ValueText ?? "", syntax.Arity, syntax.Kind(), syntax);
```

Comment:
>syntax.Identifier.ValueText ?? "" [](start = 39, length = 33)

Same comment as in another PR. This doesn't feel right.
Same comment a few lines below

Reply:
This is fixed now.